doc/rpc/components/schemas/Transaction.json

```diff
@@ -0,0 +1,73 @@
+{
+  "Transaction": {
```

Comment:
I think we likely will need `Transaction` and `TransactionRemasc`, to take into account that the 3 `v,r,s` fields are null/undefined/zero in this RSK-specific type of tx